contracts/zero-ex/contracts/src/features/TransformERC20Feature.sol

```diff
@@ -386,13 +386,19 @@ contract TransformERC20Feature is
         view
         returns (bytes32 validCallDataHash)
     {
+        address quoteSigner = getQuoteSigner();
+        if (quoteSigner == address(0)) {
+            // If no quote signer is configured, then all calldata hashes are
+            // valid.
+            return callDataHash;
```

Comment:
This is to buy us time getting 0x-API signing infra up without breaking RFQT as-is